Add files via upload

Jamal-maker123 opened the pull request:
Hall Cross Academy is a secondary school in Doncaster, UK. All the emails has a structure of:

(first four letters of last name)(4 numbers)@hallcrossacademy.co.uk

But since Hall Cross is a part of the help trust it will soon be changed to:

(first four letters of last name)(4 numbers)@hallcross.elp.org.co.uk

philipto commented:
@Jamal-maker123 hallcrossacademy.txt I am sorry, I have to close the request without merging, because you made a mistake in a request and did not specify any filepath. Please ensure that you are familiar with [this repository rules](https://github.com/JetBrains/swot?tab=readme-ov-file#swot) and make a new correct request. Thank you for your understanding!

Jamal-maker123 commented:
I am new to git hub so can you tell me what I did wrong and how to fix it.
What file do I need to upload because I already put hallcrossacademy.txt

On Thu, 13 Feb 2025, 08:13 Philip Torchinsky, ***@***.***>
wrote:

> @Jamal-maker123 <https://github.com/Jamal-maker123> hallcrossacademy.txt
> I am sorry, I have to close the request without merging, because you made a
> mistake in a request and did not specify any filepath. Please ensure that
> you are familiar with this repository rules
> <https://github.com/JetBrains/swot?tab=readme-ov-file#swot> and make a
> new correct request. Thank you for your understanding!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JetBrains/swot/pull/27244#issuecomment-2655822065>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BPN7M5XDVCYIX6B2FW5ITML2PRH3XAVCNFSM6AAAAABXAHS4RWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJVHAZDEMBWGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: philipto]*philipto* left a comment (JetBrains/swot#27244)
> <https://github.com/JetBrains/swot/pull/27244#issuecomment-2655822065>
>
> @Jamal-maker123 <https://github.com/Jamal-maker123> hallcrossacademy.txt
> I am sorry, I have to close the request without merging, because you made a
> mistake in a request and did not specify any filepath. Please ensure that
> you are familiar with this repository rules
> <https://github.com/JetBrains/swot?tab=readme-ov-file#swot> and make a
> new correct request. Thank you for your understanding!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JetBrains/swot/pull/27244#issuecomment-2655822065>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BPN7M5XDVCYIX6B2FW5ITML2PRH3XAVCNFSM6AAAAABXAHS4RWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJVHAZDEMBWGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Aleem Akhtar 

Hall Cross Academy
Upper School: Thorne Road, Doncaster, DN1 2HY
​Lower School: St Michael's Road, Doncaster, DN4 5LU
​
​T: 01302 320626
Think Wellbeing! This email was sent at a time convenient for me – please do not feel obliged to respond outside normal working hours.
Exceed Learning Partnership is registered in England and Wales under company number 10660150. Registered Office: Edlington Lane, Edlington, Doncaster, DN12 1PL.
​
​This email and any attachments may contain confidential information and are intended solely for the recipient. If you are not the intended recipient, please notify the sender immediately and delete this email. Any personal data contained within is subject to the UK GDPR, and recipients are responsible for handling it in compliance with data protection laws. For more information on our policies and procedures, including our privacy notice and data protection policies, please follow this link: [Data Protection and Privacy Policy]. Exceed Learning Partnership makes every effort to ensure emails do not contain malicious content; however, we accept no liability for any damage caused by malicious content transmitted via email.

philipto commented:
@Jamal-maker123 I added the entry for @hallcross.elp.org.co.uk.
If you need to add one more for @hallcrossacademy.co.uk, please make a request with the file `lib/domains/uk/co/hallcrossacademy.txt`. Thank you!

Jamal-maker123 commented:
Thank you so much

On Thu, 13 Feb 2025, 17:12 Philip Torchinsky, ***@***.***>
wrote:

> @Jamal-maker123 <https://github.com/Jamal-maker123> I added the entry for
> @hallcross.elp.org.co.uk.
> If you need to add one more for @hallcrossacademy.co.uk, please make a
> request with the file lib/domains/uk/co/hallcrossacademy.txt. Thank you!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JetBrains/swot/pull/27244#issuecomment-2657249828>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BPN7M5SEGGQPEYUVY4HK6AD2PTHARAVCNFSM6AAAAABXAHS4RWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJXGI2DSOBSHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: philipto]*philipto* left a comment (JetBrains/swot#27244)
> <https://github.com/JetBrains/swot/pull/27244#issuecomment-2657249828>
>
> @Jamal-maker123 <https://github.com/Jamal-maker123> I added the entry for
> @hallcross.elp.org.co.uk.
> If you need to add one more for @hallcrossacademy.co.uk, please make a
> request with the file lib/domains/uk/co/hallcrossacademy.txt. Thank you!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JetBrains/swot/pull/27244#issuecomment-2657249828>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BPN7M5SEGGQPEYUVY4HK6AD2PTHARAVCNFSM6AAAAABXAHS4RWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJXGI2DSOBSHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Aleem Akhtar 

Hall Cross Academy
Upper School: Thorne Road, Doncaster, DN1 2HY
​Lower School: St Michael's Road, Doncaster, DN4 5LU
​
​T: 01302 320626
Think Wellbeing! This email was sent at a time convenient for me – please do not feel obliged to respond outside normal working hours.
Exceed Learning Partnership is registered in England and Wales under company number 10660150. Registered Office: Edlington Lane, Edlington, Doncaster, DN12 1PL.
​
​This email and any attachments may contain confidential information and are intended solely for the recipient. If you are not the intended recipient, please notify the sender immediately and delete this email. Any personal data contained within is subject to the UK GDPR, and recipients are responsible for handling it in compliance with data protection laws. For more information on our policies and procedures, including our privacy notice and data protection policies, please follow this link: [Data Protection and Privacy Policy]. Exceed Learning Partnership makes every effort to ensure emails do not contain malicious content; however, we accept no liability for any damage caused by malicious content transmitted via email.

Jamal-maker123 commented:
Thank you for putting that one but my school email didn't change yet so if
you don't remove that but one can you please put the one I put for ____
***@***.***

You are really helping thank you so much

On Thu, 13 Feb 2025, 20:43 Akhtar, Aleem, ***@***.***>
wrote:

> Thank you so much
>
> On Thu, 13 Feb 2025, 17:12 Philip Torchinsky, ***@***.***>
> wrote:
>
>> @Jamal-maker123 <https://github.com/Jamal-maker123> I added the entry
>> for @hallcross.elp.org.co.uk.
>> If you need to add one more for @hallcrossacademy.co.uk, please make a
>> request with the file lib/domains/uk/co/hallcrossacademy.txt. Thank you!
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/JetBrains/swot/pull/27244#issuecomment-2657249828>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BPN7M5SEGGQPEYUVY4HK6AD2PTHARAVCNFSM6AAAAABXAHS4RWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJXGI2DSOBSHA>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>> [image: philipto]*philipto* left a comment (JetBrains/swot#27244)
>> <https://github.com/JetBrains/swot/pull/27244#issuecomment-2657249828>
>>
>> @Jamal-maker123 <https://github.com/Jamal-maker123> I added the entry
>> for @hallcross.elp.org.co.uk.
>> If you need to add one more for @hallcrossacademy.co.uk, please make a
>> request with the file lib/domains/uk/co/hallcrossacademy.txt. Thank you!
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/JetBrains/swot/pull/27244#issuecomment-2657249828>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BPN7M5SEGGQPEYUVY4HK6AD2PTHARAVCNFSM6AAAAABXAHS4RWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJXGI2DSOBSHA>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

Aleem Akhtar 

Hall Cross Academy
Upper School: Thorne Road, Doncaster, DN1 2HY
​Lower School: St Michael's Road, Doncaster, DN4 5LU
​
​T: 01302 320626
Think Wellbeing! This email was sent at a time convenient for me – please do not feel obliged to respond outside normal working hours.
Exceed Learning Partnership is registered in England and Wales under company number 10660150. Registered Office: Edlington Lane, Edlington, Doncaster, DN12 1PL.
​
​This email and any attachments may contain confidential information and are intended solely for the recipient. If you are not the intended recipient, please notify the sender immediately and delete this email. Any personal data contained within is subject to the UK GDPR, and recipients are responsible for handling it in compliance with data protection laws. For more information on our policies and procedures, including our privacy notice and data protection policies, please follow this link: [Data Protection and Privacy Policy]. Exceed Learning Partnership makes every effort to ensure emails do not contain malicious content; however, we accept no liability for any damage caused by malicious content transmitted via email.
